torchvision/prototype/transforms/_transform.py

```diff
@@ -0,0 +1,35 @@
+import functools
+from typing import Any, Dict, Optional
+
+from torch import nn
+from torchvision.prototype.utils._internal import apply_recursively
+from torchvision.utils import _log_api_usage_once
+
+from .functional._utils import Dispatcher
+
+
+class Transform(nn.Module):
+    _DISPATCHER: Optional[Dispatcher] = None
+
+    def __init__(self) -> None:
+        super().__init__()
+        _log_api_usage_once(self)
+
+    def get_params(self, sample: Any) -> Dict[str, Any]:
```

Comment:
Why is this public?

Reply:
No particular reason other than this is also public on our current stable transforms. I'm ok with making it private.